tests/unit/src/test/scala/tests/AddPackageSlowSuite.scala

```diff
@@ -0,0 +1,88 @@
+package tests
+
+import java.nio.file.Files
+
+import scala.meta.internal.metals.RecursivelyDelete
+import scala.meta.internal.metals.MetalsEnrichments._
+
+object AddPackageSlowSuite extends BaseSlowSuite("add package") {
```

Comment:
Please also rename to `add-package` - we don't usually include spaces in the test names.